src/main/java/com/chessmates/repository/RdsQueryExecutor.groovy

```diff
@@ -103,14 +103,10 @@ class RdsQueryExecutor implements QueryExecutor {
                 outputObject
         }
 
-        //lambda placeholder
-        def populateNone = {
 
-        }
-
-        //we only populate if it was a select query inwhich case it returns a result set
-        def returnValue = item instanceof ResultSet ? populateOutputObject(item) : populateNone
+        //we only populate if it was a select query in which case it returns a result set
+        if(!(item instanceof ResultSet)) return
```

Comment:
Confused as to how this executes for insert queries? The `stmt` above is a SELECT statement?